api/v1beta1/webhook_suite_test.go

```diff
@@ -28,7 +28,9 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
-	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	admissionv1 "k8s.io/api/admission/v1"
```

Comment:
Are we dividing the imports this way on purpose?

Reply:
Removed file